docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -33,17 +33,14 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **ORDER BY**
   - **LIKE**
```

Comment:
Now would be a good time to identify that LIKE and ORDER BY are only supported with Elasticsearch as a Visibility store.

@aarohib ?

Reply:
Added a comment about this. Waiting for @aarohib 's input

Reply:
Let's remove `LIKE` and `%` from everywhere in the doc. It doesn't work the way most people expect it to work and everything which it does can be achieved with simple `=` operator.